.gitignore

```diff
@@ -13,6 +13,7 @@ logs/
 *.log
 *.json
 out/
+FlareBot/
```

Comment:
Yegh we can't add this to the gitignore because then it ignore everything because of our path

Reply:
This is why travis failed

Reply:
😕

Reply:
Ik it's a pain, but until git gets a better gitignore system nothing we can do

Reply:
Actually it wasn't even, it's  just I didn't update gradle with my JDA and I forgo to commit a class